docs/manual/contribute.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to WECS
+
+WECS is in very early dev phase, and we'd love an ysupport we can get.
```

Comment:
"we'd love any support" instead of "we'd love an ysupport"

Reply:
Updated.